pkgs/by-name/gz/gz-utils/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  gz-cmake,
+  spdlog,
+}:
+
+let
+  core_version = "3.1.1";
+  tag_version = "gz-utils${lib.versions.major core_version}_${core_version}";
+  core_address = "https://";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-utils";
+  version = core_version;
+
+  src = fetchFromGitHub {
+    owner = "gazebosim";
+    repo = "gz-utils";
+    tag = tag_version;
+    hash = "sha256-fYzysdB608jfMb/EbqiGD4hXmPxcaVTUrt9Wx0dBlto=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gz-cmake
+    spdlog
+  ];
+
+  meta = {
+    description = "General purpose utility classes and functions for the Gazebo libraries";
+    homepage = "${core_address}gazebosim.org/home";
+    changelog = "${core_address}github.com/gazebosim/${finalAttrs.pname}/blob/${tag_version}/Changelog.md";
```

Creating an attr just for https makes the code unnecessary hard to read, also using `finalAttrs.src.tag` would be nicer to using `"gz-utils${lib.versions.major finalAttrs.version}_${finalAttrs.version}"` in fetchFromGitHub would be nicer.

Note: only https part is blocking for me.
```suggestion
    homepage = "https://gazebosim.org/home";
    changelog = "https://github.com/gazebosim/${finalAttrs.pname}/blob/${finalAttrs.src.tag}/Changelog.md";
```

---

done